### DISCUSSION TITLE
[Bug]: Video Preview not generating.

### DISCUSSION THREAD

**Discussion by alexmf23** [pimcore/pimcore#11876] [Q&A / Support] [answered]
### Expected behavior

I want to be able to preview uploaded videos.

### Actual behavior

After I upload videos, image preview is generated, but if i want to preview the video it gets stuck on " The preview for this video is currently in progress. Please wait ... ".
<img width="816" alt="Screenshot 2022-04-07 at 11 33 22" src="https://user-images.githubusercontent.com/63473281/162186925-95865c25-a866-42be-9856-f5d1769cbc9d.png">

### Steps to reproduce

Regular Pimcore V10.3.4 Install on Centos 7 running php 8.1

**Comment by jdreesen** [comment]
Are you using a remote storage for your assets? Then it's maybe fixed by #11862

**Comment by alexmf23** [comment]
> Are you using a remote storage for your assets? Then it's maybe fixed by #11862

No, I am not.

**Comment by brusch** [comment]
@alexmf23 can you please check if #11862 fixes your issue? Thanks!

**Comment by brusch** [comment]
Please also ensure you have the maintenance script running: 
https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html#page_5-Maintenance-Cron-Job

Most of the time, people forget to set this up properly 🙃

**Comment by alexmf23** [comment]
> @alexmf23 can you please check if #11862 fixes your issue? Thanks!

This didn't fixed the problem.

> Please also ensure you have the maintenance script running: https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html#page_5-
Maintenance-Cron-Job
> 
> Most of the time, people forget to set this up properly 🙃
<img width="2300" alt="Screenshot 2022-04-08 at 11 43 12" src="https://user-images.githubusercontent.com/63473281/162411487-a67e209c-8fc8-4aea-ba01-df850ab1feaa.png">

Cron jobs are running good.

**Comment by brusch** [comment]
What about the system requirements, is FFMPEG installed? 
Maybe try to run `messenger:consume` with `-vv` to see if there are any issues.

**Comment by alexmf23** [comment]
> What about the system requirements, is FFMPEG installed? Maybe try to run `messenger:consume` with `-vv` to see if there are any issues.

FFMPEG is installed. 
<img width="728" alt="Screenshot 2022-04-08 at 12 30 42" src="https://user-images.githubusercontent.com/63473281/162418974-ff7f5b1d-edc9-4756-b5b5-d4ba3ab8a5da.png">

And messenger:cosumed runned with -vv doesnt throw any errors.

**Comment by brusch** [comment, marked as answer]
@alexmf23 can you see that `messenger::consume` receives the message and starts processing?

**Reply by alexmf23** [reply]
> @alexmf23 can you see that `messenger::consume` receives the message and starts processing?

Yes it started processing. 
Problem is solved, in the end I think #11862 solved the problem.
